### ISSUE TITLE
Installing tensorflow with both conda and source

### ISSUE THREAD

**Issue by assasins353** [tensorflow/tensorflow#20427]
Hi everyone i'm new at tensorflow i have recently installed it with the package manager conda but i was having a warning message saying it was better install it from the source , so after install it from the source here is ths situation 
1/ when I use import tensorflow without using conda (from the source) it doesn't work saying that there is no mudule called tensorflow 
2/ when i use conda (source activate project_name) tensorflow work fine and without any warning anymore
Is it possible to use tensorflow from the source (without conda ) with conda still installed or should i remove conda and how ? 
when i try sudo pip3 install /tmp/tensorflow_pkg/tensorflow-1.8.0-py2-none-any.whl consol says that tensorflow is perfectly installed 
my configuration:
 ubuntu 18.04
tensorflow 1.8.0 for conda and 1.9.0cr0 for the source
pyhton 3.6.5
and i use cpu configuration not the gpu one

**Comment by marshalhayes** [comment]
Hi @assasins353. I'm not sure why `conda` would warn you that installing from source is better. Anyway, you can certainly install TensorFlow from source or through `conda`. Neither of which are recommended. 

For `conda`, TensorFlow 1.8.0 is provided in the `conda-forge` channel. You can install TensorFlow 1.8.0 by `conda install -c conda-forge tensorflow==1.8.0`. 

Now, if you are trying to install TensorFlow with `pip` in a `conda` environment, this may cause issues. I noticed when I had two versions of TensorFlow (one install through pip, and the other through conda) that the conda installation would take precendent over the pip installation. This means that `tf.__version__` would return the version installed through `conda`. 

It's easy to avoid all this by creating virtual environments. Follow the recommended [TensorFlow installation documentation](https://www.tensorflow.org/install/). If you want to install TensorFlow from source and with `conda`, you should create a virtual environment for the source installation. See the [virtualenv documentation](https://virtualenv.pypa.io/en/stable/userguide/) to create a virtual environment. The other could be kept in your main base conda environment.

**Comment by aselle** [comment]
@assasins353, I'm closing as @marshalhayes' response is pretty comprehensive. In short this is working as we expect.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @aselle: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by VanitarNordic** [comment]
if you use Anaconda, you can use  `conda install -c hesi_m tensorflow ` to install **tensorflow 1.9.0** for CPU 

if you specifically use Keras also, you can use  `conda install -c hesi_m keras` which installs **Keras 2.2.2** in accompany with everything including tensorflow 1.9.0

It is better to make a fresh anaconda environment for your experiments

**Comment by tensorflowbutler** [comment]
Nagging Assignee @aselle: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @aselle: It has been 30 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @aselle: It has been 45 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @aselle: It has been 60 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @aselle: It has been 75 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @aselle: It has been 90 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @aselle: It has been 105 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @aselle: It has been 120 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by jvishnuvardhan** [comment]
Closing this out since I understand it to be resolved, but please let me know if I'm mistaken.Thanks!
